README.md

```diff
@@ -1,2 +1,50 @@
 # 3D-Vac
 Repository of the eScience Center and Radboudumc "Personalized cancer vaccine design through 3D modelling boosted geometric learning" collaboartive project (OEC 2021).
+
+## Repository structure
+
+I used [Cookiecutter Data Science](https://drivendata.github.io/cookiecutter-data-science/#starting-a-new-project) to create this project structure. The following is the directory structure:
+
+```
+├── LICENSE
+├── README.md          <- The top-level README for developers using this project.
+├── data
+│   ├── external       <- Data from third party sources.
```

Comment:
├── external or (raw)  <- BA and EL text data (csv)
├── external_processed <- csv files ready for the modelling
├── blast_dbs  <- Blast databases and MSAs to generate them
├── 3D_models
├── PSSMs
├── Mapped_PSSMs
├── Aligned_3D_models
├── HDF5
├── NN_models

Reply:
Seems great! I added some subfolders:
├── raw
├── blas_dbs
├── hdf5
├── 3d_models
  ├── aligned
  ├── unaligned
├── external
  ├── processed
  ├── unprocessed
├── pssm
  ├── mapped
  ├── unmapped

I think they are even clearer to distinguish in this way, but let me know if you like it. For NN_models, there is an entire folder in the main dir called `models`, if you mean train networks and serialized models. Is that good?

Reply:
Looks good! And regarding models: I think it is good to keep trained models and model predictions separate from the data, but as long as it is consistent I'm fine with anything that makes sense.